[2025-04-01T18:48:07.719+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T18:48:07.728+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.task1 manual__2025-04-01T18:48:06.651199+00:00 [queued]>
[2025-04-01T18:48:07.732+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.task1 manual__2025-04-01T18:48:06.651199+00:00 [queued]>
[2025-04-01T18:48:07.732+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-04-01T18:48:07.737+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task1> on 2025-04-01 18:48:06.651199+00:00
[2025-04-01T18:48:07.742+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'task1', 'manual__2025-04-01T18:48:06.651199+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpdqpl8vn9']
[2025-04-01T18:48:07.743+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask task1
[2025-04-01T18:48:07.743+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5241) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-01T18:48:07.744+0000] {standard_task_runner.py:72} INFO - Started process 5242 to run task
[2025-04-01T18:48:07.771+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.task1 manual__2025-04-01T18:48:06.651199+00:00 [running]> on host 7d980c7fabe2
[2025-04-01T18:48:07.814+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='grypton' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T18:48:06.651199+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T18:48:06.651199+00:00'
[2025-04-01T18:48:07.815+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T18:48:07.829+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-01T18:48:07.830+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello World!!!, this is the very first dag'"]
[2025-04-01T18:48:07.836+0000] {subprocess.py:99} INFO - Output:
[2025-04-01T18:48:07.848+0000] {subprocess.py:106} INFO - Hello World!!!, this is the very first dag
[2025-04-01T18:48:07.848+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-01T18:48:07.867+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T18:48:07.867+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=task1, run_id=manual__2025-04-01T18:48:06.651199+00:00, execution_date=20250401T184806, start_date=20250401T184807, end_date=20250401T184807
[2025-04-01T18:48:07.922+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T18:48:07.940+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-01T18:48:07.941+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
